Sources/SwiftGen/Config/Config+Run.swift

```diff
@@ -29,18 +29,23 @@ extension Config {
     }
   }
 
-  private func run(cmd: ParserCLI, entry: ConfigEntry, verbose: Bool) throws {
+  private func run(
+    cmd: ParserCLI,
+    entry: ConfigEntry,
+    verbose: Bool,
+    logger: @escaping (LogLevel, String) -> Void
+  ) throws {
     var entry = entry
 
     entry.makingRelativeTo(inputDir: inputDir, outputDir: outputDir)
```

Comment:
_(note for separate PR: we should rename that method to `makeRelativeTo` or something, that current naming feels odd)_